style.css

```diff
@@ -0,0 +1,151 @@
+/* CSS files add styling rules to your content */
+
+body {
+  font-family: "Benton Sans", "Helvetica Neue", helvetica, arial, sans-serif;
+  margin: 2em;
+}
+
+h1 {
+  color: #373fff;
+}
+
+.code-area {
+  width: 100%;
+  min-height: 15em;
+  border: solid 1px;
+  padding: 5px;
+  box-shadow: 1px 4px 2px -1px rgba(200, 200, 250, 0.7); 
+  font-size: 14px;
+  resize: none;
+  font-family: monospace;
+}
+
+.code-area:focus {
+  outline: none;
+}
+
+.column-container {
+  display: flex;
+}
+
+.column {
+  width: 40%;
+}
+
+std-toast {
+  font-size: 36px;
+}
+
+#toast-1 {
```

Comment:
The duplication of styles here vs. in the textareas seems bad.

Can we instead remove all of these, change the textareas to use ID selectors (instead of `std-toast`), and synchronize the `<textarea>`s with `<style>` elements, like you do for the sandbox, but everywhere?

Reply:
That's doable and I considered it, but it would mean that when you put them in the sandbox you'd have to change it from referring to `std-toast` to `#toast-#`. Definitely agree that this is weird behavior, but that's not the best behavior either.

Reply:
You know what, I think I can add some JS that fixes that, lemme do that instead